study-builder/studies/testboston/study-workflows.conf

```diff
@@ -51,6 +51,15 @@
         "activityCode": ${id.act.address}
       },
       "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": ${id.act.symptoms_survey},
+          "expression": """
+            user.studies["testboston"].forms["ADDRESS"].isStatus("COMPLETE")
+            && user.studies["testboston"].forms["SYMPTOMS_SURVEY"].hasInstance()
```

Comment:
I was thinking there might be an issue here since the Workflow service will create an instance if it doesn't exist. But I see you have solved this this `hasInstance()` check here, so we should be good for now.